lib/Ohmbrewer_Screen.h

```diff
@@ -5,9 +5,18 @@
 #undef min
 #undef max
 #undef swap
+
+//Below definitions are for the touch screen
+#define XP A1
+#define YP A0
+#define XM A7
+#define YM A2
+
```

Comment:
Change the `#define`'s for XP, YP, XM, and YM into static consts as we do for the other consts.